.travis.yml

```diff
@@ -19,21 +22,26 @@ before_install:
   # There's a bug in the Trusty images where this gives a syntax error
   # See https://github.com/travis-ci/travis-ci/issues/8906
   - sudo mv /opt/jdk_switcher/jdk_switcher.sh /tmp
-
-  - wget -q "http://download.nuohub.org/nuodb-ce_${NUO_VERSION}_amd64.deb" --output-document=/var/tmp/nuodb.deb
+  - wget -q "$NUO_DOWNLOAD/supportedversions.txt" --output-document=/tmp/supportedversions.txt
+  - wget -q "$NUO_DOWNLOAD/${NUO_PREFIX}_$(tail -n1 /tmp/supportedversions.txt)_amd64.deb" --output-document=/var/tmp/nuodb.deb
   - sudo dpkg -i /var/tmp/nuodb.deb
+  - echo "NuoDB Version:" && $NUODB_HOME/bin/nuodb --version
 
 before_script:
-  - if [ "$NUO_ADD_DOMAIN_PASS" = yes ]; then printf '\ndomainPassword = bird\n' | sudo tee -a "$NUODB_HOME"/etc/default.properties >/dev/null; fi
-  - if [ "$NUO_START_AGENT" = yes ]; then sudo service nuoagent start; fi
-  - $NUODB_HOME/bin/nuodbmgr --broker localhost --password bird --command "start process sm host localhost database test archive /var/tmp/nuodb initialize true"
-  - sleep 5
-  - $NUODB_HOME/bin/nuodbmgr --broker localhost --password bird --command "start process te host localhost database test options '--dba-user dba --dba-password goalie'"
-  - sleep 5
+  - sudo -n $NUODB_HOME/etc/nuoadmin tls $NUO_SET_TLS
+  - sudo -n systemctl start nuoadmin
+  # In a future state, this could be done by the test harness using the driver itself as it is in the python repo
+  - $NUODB_HOME/bin/nuocmd create archive --db-name test --dba-user dba --dba-password goalie --archive-path /var/tmp/nuodb --server-id `$NUODB_HOME/bin/nuocmd get servers | awk -F, '{print $4}' | awk -F= '{print $2}'`
+  - $NUODB_HOME/bin/nuocmd create database --db-name test --dba-user dba --dba-password goalie
+  - $NUODB_HOME/bin/nuocmd start process --engine-type TE --db-name test --this-server
 
 script:
   - xbuild /p:TargetFrameworkProfile="" /p:Configuration=Release ADO.NET.Mono.sln
   - mono ConsoleSample/bin/Release/ConsoleSample.exe test
   - mono NUnitTestProject/bin/Release/NUnitTestProject.exe
   - dotnet build ADO.NET.Core.sln
   - dotnet ConsoleSample/bin/Release/netcoreapp2.1/ConsoleSampleCore.dll test
+
+after_failure:
+  - cat /var/log/nuodb/nuoadmin.log*
+  - grep ssl /etc/nuodb/nuoadmin.conf
```

Comment:
Please add a newline at the end of the file.  You can configure your editor for this typically.